projects/canopy/src/lib/forms/date/date-field.notes.ts

```diff
@@ -17,7 +17,8 @@ Import the Date Input Module into your application:
 and in your HTML:
 
 ~~~html
-<lg-date-field formControlName="date">
+<form [formGroup]="form" (ngSubmit)="..." #validationForm="ngForm">
```

Comment:
I wonder if we could add a HTML comment as well, to make validation thing easy to notice at a glance. E.g.

```suggestion
<!-- It's important to include \`ngForm\` on the form tag for validation to work properly  -->
<form [formGroup]="form" (ngSubmit)="..." #validationForm="ngForm">
```

What do you think?

Reply:
Yes, I'll copy the entire comment in there too. The submit button within the form tag is also equally important so it's worth adding that too 👍

Reply:
@owensgit the comments get removed when deploying the code so. I've added a bold note under the html.